index.js

```diff
@@ -60,10 +60,12 @@ module.exports = function(config) {
         logger = makeLogger(name, name);
         context.logger = logger;
         var getConnection = require('./lib/api/get_connection')(context);
+        var events = require('./lib/api/events');
         context.foundation = {
             makeLogger: makeLogger,
             startWorkers: require('./lib/api/start_workers')(context),
-            getConnection: getConnection
+            getConnection: getConnection,
+            events: events
```

Comment:
If it were simply a function that returned the event emitter I wouldn't be commenting on it as long as it was named reasonably. Just remember these are APIs used by end users which means they're going to be writing code that utilizes them so as capabilities are added it needs to be consistent and logical.